BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/IdentifyFragment.java

```diff
@@ -44,6 +45,14 @@ public void onClick(View view) {
                 transaction.addToBackStack(null);
 
                 transaction.commit();
+
+
+              // the code below was causing the app to stop working so I'm commenting it out for now
+              //  TextView beeSpeciesID = (TextView) view.findViewById(R.id.speciesID);
```

Comment:
This won't work because there is no TextView on the identify fragment with the id "speciesID". You did add some strings with the id "speciesID" and "scientificID" but that's not how findViewById works. findViewById only checks the appropriate layout xml file for android:id fields, it doesn't search through strings.

Did you mean to put this code in review fragment? You shouldn't be changing the identify fragment for this issue